src/app/globals.css

```
@@ -10,6 +10,25 @@
+  --color-mc-fg-muted: #5E5774;
+  --color-mc-uv: #7C7CFF;
+  --color-mc-uv-hi: #9D9CFF;
+  --color-mc-pink: #FF7AD8;
+  --color-mc-green: #3DDC97;
+  --color-mc-amber: #FFB454;
+  --color-mc-red: #FF6469;
+  --color-mc-cyan: #5DD9FF;
+
+  --font-mc-sans: var(--font-geist), 'Geist', 'Inter', system-ui, sans-serif;
+  --font-mc-mono: var(--font-mono), 'JetBrains Mono', 'Geist Mono', ui-monospace, monospace;
+  --font-mc-serif: var(--font-display), 'Instrument Serif', Georgia, serif;
```

Stylelint (17.12.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)